Library/Homebrew/os/mac/sdk.rb

```diff
@@ -0,0 +1,57 @@
+require "os/mac/version"
+
+module OS
+  module Mac
+    class SDK
+      attr_reader :version, :path
+
+      def initialize version, path
+        @version = OS::Mac::Version.new version
```

Comment:
FWIW when Jack renamed this to `OS::Mac`, the intention was that we'd keep using `MacOS` indefinitely. I used `OS::Mac` within this file because the `MacOS` synonym might not be defined when this file is loaded, but it definitely is within the `OS::Mac` module itself.